src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -0,0 +1,224 @@
+    und Ersatzzeiten.
+
+    Parameters
+    ----------
+    entgeltp
+        See basic input variable :ref:`entgeltp <entgeltp>.
+    m_beitragszeit
+        See basic input variable :ref:`m_beitragszeit <m_beitragszeit>.
+
+    Returns
+    -------
+    Average Entgeldpunkte per year
+
+    """
+    out = entgeltp / (m_beitragszeit / 12)
+    return out
+
+
+def entgeltp_erwerbsm_rente(
+    entgeltp: float,
+    durchschnittliche_entgeltp: float,
+    erwerbsm_rente_params: dict,
+    geburtsjahr: int,
+    jahr_renteneintr: int,
+) -> float:
+    """Calculating Entgeltpunkte for Erwerbsminderungsrente
+    (pension for reduced earning capacity)
+
+    Parameters
+    ----------
+    entgeltp
+        See basic input variable :ref:`entgeltp <entgeltp>
+    durchschnittliche_entgeltp
+        See :func:`durchschnittliche_entgeltp`.
+    erwerbsm_rente_params
+        See params documentation :ref:`erwerbsm_rente_params <erwerbsm_rente_params>.
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>.
+    jahr_renteneintr
+        See basic input variable :ref:`jahr_renteneintr <jahr_renteneintr>.
+
+    Returns
+    -------
+    Final Entgeltpunkte for Erwerbsminderungsrente
+
+    """
+
+    out = entgeltp + (
+        (
+            erwerbsm_rente_params["zurechnungszeitsgrenze_in_m"] / 12
+            - (jahr_renteneintr - geburtsjahr)
+        )
+        * durchschnittliche_entgeltp
+    )
+
+    return out
+
+
+def rentenfaktor_erwerbsm_rente(
+    teilw_erwerbsm_rente: bool,
+) -> float:
+    """Checking Rentenfaktor for Erwerbsminderungsrente
+    (pension for reduced earning capacity)
+    based on input variable teilw_erwerbsm_rente
+
+    Als Input Variable
+    Parameters
+    ----------
+    teilw_erwerbsm_rente
+        See basic input variable :ref:`teilw_erwerbsm_rente <teilw_erwerbsm_rente>.
+
+    Returns
+    -------
+    Rentenfaktor
+
+    """
+
+    if teilw_erwerbsm_rente:
+        out = 0.5
+
+    else:
+        out = 1
+
+    return out
+
+
+def erwerbsm_rente_vor_grundr_m(
+    erwerbsm_rente_zugangsfaktor: float,
+    entgeltp_erwerbsm_rente: float,
+    rentenwert: float,
+    rentenfaktor_erwerbsm_rente: float,
+    anspruch_erwerbsm_rente: bool,
+) -> float:
+    """Calculating the Erwerbsminderungsrente pension claim
+
+    Parameters
+    ----------
+    ges_rente_zugangsfaktor
+        See :func:`ges_rente_zugangsfaktor`.
+    entgeltp_fuer_erwerbsm_rente
+        See :func:`entgeltp_fuer_erwerbsm_rente`.
+    rentenwert
+        See :func:`rentenwert`.
+    rentenfaktor_fuer_erwerbsm_rente
+        See :func:`rentenfaktor_fuer_erwerbsm_rente`.
+    rentner
+        See basic input variable :ref:`rentner <rentner>.
+    Returns
+    -------
+    Erwerbsminderungsrente pension claim
+
+    """
+
+    if anspruch_erwerbsm_rente:
+        out = (
+            entgeltp_erwerbsm_rente
+            * erwerbsm_rente_zugangsfaktor
+            * rentenwert
```

Comment:
How is the distinction of Rentenwert Ost/West implemented in GETTSIM? I think technically we need two separate variables for pension points that were earned in the West and East respectively as these are each multiplied by a different Rentenwert.